routes/web.php

```diff
@@ -18,6 +18,14 @@
 |--------------------------------------------------------------------------
 */
 Route::get('/', ['as' => 'frontpage', 'uses' => 'WelcomeController@index']);
+Route::post('/getzip', ['as' => 'getzip', 'uses' => 'Document\BulkDownloadController@buildzip']);
```

Comment:
I would probably call it something like `documents.bulk.download` and on the path I would do something like `/documents-download`. `/getzip` is too generic